app/controllers/PlutoController.scala

```diff
@@ -37,6 +37,12 @@ class PlutoController(
     Ok(Json.toJson(plutoProjects))
   }
 
+  def deleteCommission(id: String) = APIHMACAuthAction {
+    stores.plutoProjectStore.deleteByCommissionId(id)
```

Comment:
Should we check the return result of these two to catch errors?

Reply:
`Scanamo.delete` returns a `DeleteItemResult`, which, looking at [the docs](https://docs.aws.amazon.com/sdkfornet1/latest/apidocs/html/T_Amazon_DynamoDB_Model_DeleteItemResult.htm), returns on success or failure. I don't think we necessarily care if we try to delete a non-existing commission...

Reply:
Who re-sends the message?

Reply:
Pluto would.

Reply:
OK cool, so it checks to see if the deletion has actually worked and retries accordingly?